commerce_coordinator/apps/titan/exceptions.py

```diff
@@ -2,7 +2,13 @@
 from rest_framework.exceptions import APIException
 
 
-class PaymentNotFond(APIException):
+class PaymentNotFound(APIException):
     status_code = 404
     default_detail = 'Requested payment not found. Please make sure you are passing active payment number.'
     default_code = 'payment_not_found'
+
+
+class NoActiveOrder(APIException):
+    status_code = 404
```

Comment:
I think [Spec](https://github.com/edx/commerce-coordinator-private/blob/main/docs/api-spec/theseus-api-spec.md#status-code-6) needs to be fixed. I can't see this error there.

Reply:
https://github.com/edx/commerce-coordinator-private/blob/main/docs/api-spec/versions/theseus-api-spec_2023-06-19.md#error-7
Will  also update spec to add it to the frontend-app-payment call